docs/topics_5/web-execution-config.adoc

```diff
@@ -1,14 +1,14 @@
 // Module included in the following assemblies:
 // * docs/web-console-guide_5/master.adoc
 [id='analysis_configuration_{context}']
-= Configuring the Analysis
+= Configuring the analysis
 
 Configure the settings for the analysis, such as the transformation path, packages, and other advanced options.
 
-.Configuring Analysis
-image::web-configure-analysis.png[Configure Analysis]
+.Configuring analysis
+image::web-configure-analysis.png[Configure analysis]
```

Comment:
Configuring an analysis. Change both image caption and image name

Reply:
Done.